packages/web-app-files/tests/mixins/collaborators.spec.js

```diff
@@ -0,0 +1,30 @@
+import CollaboratorsMixins from '../../src/mixins/collaborators'
+import { shallowMount } from '@vue/test-utils'
+
+describe('mixins/collaborators', () => {
+  describe('collaboratorType', () => {
+    it.each`
+      shareType    | expectedResult
+      ${0}         | ${'User'}
+      ${1}         | ${'Group'}
+      ${2}         | ${'Unknown type'}
+      ${3}         | ${'Unknown type'}
```

Comment:
this is a public link share, but I guess `Unknown type` is correct here because this in used in the collaborators/people list and the public links are displayed separately
@kulmann is that true?